Application/MicroBooter/MicroBooter.cpp

```diff
@@ -3,7 +3,7 @@
 //  <No description>
 //
 //  Microsoft dotNetMF Project
-//  Copyright �2004 Microsoft Corporation
+//  Copyright ©2004 Microsoft Corporation

```

Comment:
Yeah, I was thinking with the vNext in mind whether there is still any tool in the chain that couldn't handle Unicode...